libs/dh/core/shell/src/lib/shell/shell.component.html

```diff
@@ -15,9 +15,18 @@
 limitations under the License.
 -->
 <watt-shell>
-  <ng-container watt-shell-sidenav> Sidenav </ng-container>
+  <ng-container watt-shell-sidenav>
+    <div class="sidenav">
+      <img
+        src="assets/logo-light.svg"
+        alt="DataHub logo"
+        class="logo--sidenav"
```

Comment:
suggestion: I think we should agree on the methodology used for naming classes to be consisten. Does this follow BEM?

Reply:
Yes this is BEM `--sidenav` is ment as a modifier to logo